src/clarity-angular/data/stack-view/_stack-view.clarity.scss

```diff
@@ -44,6 +44,7 @@ $clr-stack-font-weight: clr-getTypePropertyValueForDomElement(stackview_text, fo
         border-radius: $clr-default-borderradius;
         overflow-y: auto;
         background-color: $gray-lighter;
+        word-wrap: break-word;
```

Comment:
Shouldn't this be `break-all`? https://developer.mozilla.org/en-US/docs/Web/CSS/word-break. I dont see `break-word` as a property value. Please correct me if I am wrong.